packages/in_app_purchase/lib/src/in_app_purchase/purchase_details.dart

```diff
@@ -107,6 +109,39 @@ class PurchaseParam {
 
   /// The 'sandboxTesting' is only available on iOS, set it to `true` for testing in AppStore's sandbox environment. The default value is `false`.
   final bool sandboxTesting;
+
+  /// The 'changeSubscriptionParam' is only available on Android, for upgrading or
```

Comment:
nits: dart doc should start with a single sentence summary and an empty line between summary and description. See: https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

Reply:
Done